internal/resources/resource_project.go

```diff
@@ -173,6 +174,13 @@ func projectSchema() map[string]*schema.Schema {
 			ForceNew:    true,
 			Description: "Volume replication is enabled for the project if set.",
 		},
+
+		pBootFromSANSupport: {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
+			Description: "Boot-from-SAN feature is supported for the project if set.",
```

Comment:
"supported" or "enabled"?

Reply:
But I see the description for VolumeReplication says Enabled. That would be good to stay consistent on that. Let me reword this.